torch/_dynamo/config.py

```diff
@@ -230,6 +230,9 @@
 # trace through numpy ndarray as tensor and try to translate numpy function to torch function.
 numpy_ndarray_as_tensor = False
 
+# Uses z3 for validating the guard optimizations transformations.
+translation_validation = True
```

Comment:
Should be False by default, methinks

Reply:
cf. https://github.com/pytorch/pytorch/pull/102563#issuecomment-1574434446